src/ostree.jsx

```diff
@@ -629,7 +632,7 @@ const OStreeSource = ({ ostreeState, refreshRemotes, onChangeBranch, onChangeRem
     const Dialogs = useDialogs();
 
     const actions = [
-        <DropdownItem key="rebase"
+        <DropdownItem key="rebase" data-action="rebase"
```

Comment:
Would be nice if the key leaked through to the DOM right ;-)

Reply:
heh yea, but it's only a React detail..